charts/loki-distributed/templates/distributor/poddisruptionbudget-distributor.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if gt (int .Values.distributor.replicas) 1 }}
+{{- if and (gt (int .Values.distributor.replicas) 1) .Values.distributor.maxUnavailable }}
```

Comment:
I think directly telling the user what's wrong would be better. Like this:

```
{{- if gt (int .Values.distributor.replicas) 1 }}
{{- if not .Values.distributor.maxUnavailable }}
{{- fail "`.Values.distributor.maxUnavailable` must be specified." }}
{{- else }}
...
{{- end }}
{{- end }}
```

Reply:
Thank you @zanhsieh, makes sense, I've updated the PR.